studio/src/components/IconWithNotification.tsx

```diff
@@ -0,0 +1,66 @@
+import { cn } from "@/utils";
+import { Icon, type IconProps } from "@iconify/react";
+import type React from "react";
+
+interface IconWithNotificationProps extends IconProps {
+  notificationColor?: string;
+  notificationSize?: number;
+  notificationPosition?:
+    | "top-right"
+    | "top-left"
+    | "bottom-right"
+    | "bottom-left";
+  notificationContent?: string | number;
+  showNotification?: boolean;
+}
+
+const IconWithNotification: React.FC<IconWithNotificationProps> = ({
+  notificationColor = "bg-red-500",
+  notificationSize = 10,
+  notificationPosition = "top-right",
+  notificationContent,
+  showNotification = true,
+  ...iconProps
+}) => {
+  const getNotificationStyle = (): React.CSSProperties => {
+    const baseStyle: React.CSSProperties = {
```

Comment:
Makes me wonder why not use tailwind (for the more typical things like flex/asbsolute/alignment etc)